src/mudlet.cpp

```diff
@@ -3420,3 +3420,27 @@ void mudlet::showChangelogIfUpdated()
     updater->showChangelog();
 }
 #endif // INCLUDE_UPDATER
+
+int mudlet::getColumnCount( Host * pHost, QString & name )
+{
+    QMap<QString, TConsole *> & dockWindowConsoleMap = mHostConsoleMap[pHost];
+
+    if( dockWindowConsoleMap.contains( name ) ) {
+        return dockWindowConsoleMap[name]->console->getColumnCount();
+    }
+
+    TDebug(QColor(Qt::white),QColor(Qt::red))<<"ERROR: window doesn't exist\n" >> 0;
+    return 0;
```

Comment:
We should return `-1` here and in the same place in the other one below so that the `TLuaInterpreter` caller can produce a __nil+errorMessage__ run-time result should the window not be found.